src/components/Stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { Helmet } from 'react-helmet';
+import Card from './Card';
+import RectangleSVG from '../static/images/rectangle.svg';
+
+const url = 'https://api.rss2json.com/v1/api.json?rss_url=https://medium.com/feed/keep-informed';
+
+const Stories = () => {
+  const [mediumData, setMediumData] = useState([]);
+  useEffect(() => {
+    axios.get(url).then((response) => {
```

Comment:
do you have any additional thoughts on using axios? I know right now we usually use fetch, but is there a specific reason why you chose axios?